packages/liveblocks-react-comments/src/styles/index.css

```diff
@@ -816,27 +816,27 @@
   display: flex;
   gap: calc(0.75 * var(--lb-spacing));
   align-items: center;
-  height: var(--lb-comment-avatar-size);
+  block-size: var(--lb-comment-avatar-size);
```

Comment:
I've been thinking about languages with vertical writing systems would like to display Comments, and it feels unusual, e.g. a composer with an input that's far taller than it is wide. But if they're typing in vertically, I guess it makes sense!

Reply:
We're applying vertical-specific rules to avoid adding linting exceptions but I think for now we shouldn't recommend vertical writing modes, I don't think we can say we fully support them like we do for RTL.